### PR TITLE
feat(Task-1D): As a Project Manager, already done exercises uploaded to the git repository

### PR DESCRIPTION
**Pull request by SamuFigueirido**

### CODE REVIEW

**Review by eloydotco — APPROVED** [review]
Looks good in general, but the code formatting looks a little bit strange for me :)
for example, there is no spacing on assigment statements like `const fifteen=inventors...`
is this intended? do you use a code formatter?

File: Ejercicio4/index-START.html [hunk 1]
```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
```
Comment (gbtec-ext-jorgevillar) [inline Ejercicio4/index-START.html:1]: Change the target branch of this PR to 'development' and check it on your other branches

File: Ejercicio4/index-START.html [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+      { first: 'Sarah E.', last: 'Goode', year: 1855, passed: 1905 },
+      { first: 'Lise', last: 'Meitner', year: 1878, passed: 1968 },
+      { first: 'Hanna', last: 'Hammarström', year: 1829, passed: 1909 }
+    ];
+
+    const people = ['Beck, Glenn', 'Becker, Carl', 'Beckett, Samuel', 'Beddoes, Mick', 'Beecher, Henry', 'Beethoven, Ludwig', 'Begin, Menachem', 'Belloc, Hilaire', 'Bellow, Saul', 'Benchley, Robert', 'Benenson, Peter', 'Ben-Gurion, David', 'Benjamin, Walter', 'Benn, Tony', 'Bennington, Chester', 'Benson, Leana', 'Bent, Silas', 'Bentsen, Lloyd', 'Berger, Ric', 'Bergman, Ingmar', 'Berio, Luciano', 'Berle, Milton', 'Berlin, Irving', 'Berne, Eric', 'Bernhard, Sandra', 'Berra, Yogi', 'Berry, Halle', 'Berry, Wendell', 'Bethea, Erin', 'Bevan, Aneurin', 'Bevel, Ken', 'Biden, Joseph', 'Bierce, Ambrose', 'Biko, Steve', 'Billings, Josh', 'Biondo, Frank', 'Birrell, Augustine', 'Black, Elk', 'Blair, Robert', 'Blair, Tony', 'Blake, William'];
+
+    // Array.prototype.filter()
+    // 1. Filter the list of inventors for those who were born in the 1500's
+    const fifteen=inventors.filter(function(inventor){
```
Comment (gbtec-ext-jorgevillar) [inline Ejercicio4/index-START.html]: Use a Lambda here

File: Ejercicio4/index-START.html [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+      if(inventor.year >= 1500 && inventor.year < 1600){
+        return true;
+      }
+    });
+    console.table(fifteen);
+
+    // Array.prototype.map()
+    // 2. Give us an array of the inventors' first and last names
+    const fullNames=inventors.map(inventor => inventor.first +' '+ inventor.last);
```
Comment (gbtec-ext-jorgevillar) [inline Ejercicio4/index-START.html:43]: place whitespace before and after the '=' in all the JS code

**Review by gbtec-ext-jorgevillar — APPROVED** [review]

**Review by gbtec-ext-jorgevillar — CHANGES_REQUESTED** [review]
I wanted to reget the changes

**Review by gbtec-ext-jorgevillar — APPROVED** [review]

**Pull request merged**